sunpy/visualization/visualization.py

```diff
@@ -45,15 +48,51 @@ def axis_labels_from_ctype(ctype, unit):
     labels = {'HGLN': 'Heliographic Longitude',
               'CRLN': 'Carrington Longitude',
               'HPLN': 'Helioprojective Longitude (Solar-X)',
+              'HRLN': 'Helioprojective Position Angle',
               'SOLX': 'Heliocentric X',
 
               'HGLT': 'Latitude',
               'CRLT': 'Latitude',
               'HPLT': 'Helioprojective Latitude (Solar-Y)',
+              'HRLT': 'Helioprojective Declination',
               'SOLY': 'Heliocentric Y'}
 
     label = labels.get(ctype_short, f"{ctype}")
     if unit is not None:
         label += f' [{unit}]'
 
     return label
+
+
+def show_hpr_impact_angle(declination_axis):
+    """
+    Modify a declination axis for Helioprojective Radial plots to show impact angle
+    instead.
+
+    In order to accommodate the FITS-WCS machinery, the declination is used instead
+    of the impact angle (which is the declination plus 90 degrees).  This function
+    changes the tick-label formatter for an axis so that it adds 90 degrees before
+    rendering the labels.
+
+    Parameters
+    ----------
+    coordinate_axis : `astropy.visualization.wcsaxes.CoordinateHelper`
+        The coordinate axis for Helioprojective Radial declination (``delta``)
+
+    See Also
+    --------
+    sunpy.coordinates.HelioprojectiveRadial
+
+    Examples
+    --------
+    .. minigallery:: sunpy.visualization.show_hpr_impact_angle
+    """
+    if not isinstance(declination_axis, CoordinateHelper):
+        raise TypeError("The input should be one of the WCSAxes coordinate axes.")
+
+    old_format_method = declination_axis._formatter_locator.formatter
```

Comment:
Even with current astropy we can remove one private API access here.
```suggestion
    old_format_method = declination_axis.formatter
```